Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectListInstance.cs

```diff
@@ -123,17 +123,17 @@ public override TokenParser ProvisionObjects(Web web, ProvisioningTemplate templ
                             foreach (var fieldRef in listInfo.TemplateList.FieldRefs)
                             {
                                 var field = rootWeb.GetFieldById<Field>(fieldRef.Id);
-                                if (field != null)
+                                if (field == null) continue;

```

Comment:
Should we continue when field is not found? What if you copied inaccurate id by mistake or content type hub doesn't work as expected. You will never know that some of your `fieldrefs` are not updated (because they are wrong).